package.json

```diff
@@ -46,8 +46,8 @@
     "start": "node bin/www",
     "lint": "eslint --ext .js,.vue .",
     "test": "mocha",
-    "coverage": "c8 --check-coverage --lines 100 --branches 100 --functions 100 mocha",
-    "record-coverage": "c8 report --reporter=text-lcov > ./coverage/lcov.info",
+    "coverage": "c8 --clean mocha",
+    "record-coverage": "c8 report --reporter=text-lcov --all --exclude=test/* --exclude=.git --exclude=.eslintrc.js --exclude=.postcssrc.js --exclude=wallaby.js > ./coverage/lcov.info",
```

Comment:
Is there a way to put all this options in a file like `.c8`?

Reply:
nothing in their docs which is annoying, not read the source to see if its a hidden thing, could just put it in a `.c8` and then `$(cat .c8)` at runtime

Reply:
keep that for another PR :)